plutus-tx-plugin/test/Plugin/Coverage/9.6/coverageCode.pir.golden

```diff
@@ -102,6 +102,27 @@
       )
       equalsInteger
     )
+    (termbind
+      (nonstrict)
+      (vardecl `&&` (fun Bool (fun Bool Bool)))
```

Comment:
This shouldn't exist, surely?

Reply:
Ok, so this solution never removes the function declaration itself, only rewrites its fully saturated applications...
M.b. there are partial applications left (in which case I'd expect this binding to remain)?

Reply:
The application in this example looks saturated to me...

Reply:
Ok, this `&&` isn't the `PlutusTx.Bool.(&&)` (which is exactly the point of this PR - tell different `&&`s apart)

Reply:
Haha, good spot. Arguably that's a bug in the test